test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -154,7 +154,8 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
         uint256 balance0BeforeAlice = currency0.balanceOf(alice);
         uint256 balance1BeforeAlice = currency1.balanceOf(alice);
 
-        vm.startPrank(alice);
+        // bob can increase liquidity for alice even though he is not the owner / not approved
```

Comment:
could you add a new test for this? so just keep this test the exact same and add a new one, it could do probably even simpler asserts (ie just that alices liquyidity goes up by the exact amount) but pranks bob